src/material-examples/cdk-datepicker-overview/cdk-datepicker-overview-example.ts

```diff
@@ -0,0 +1,50 @@
+import {Component} from '@angular/core';
+import {CalendarView, DateAdapter} from '@angular/cdk/datetime';
+
+
+/** @title Basic CDK datepicker */
+@Component({
+  selector: 'cdk-datepicker-overview-example',
+  templateUrl: 'cdk-datepicker-overview-example.html',
+  styleUrls: ['cdk-datepicker-overview-example.css'],
+})
+export class CdkDatepickerOverviewExample {
+  messages: string[] = [];
+  _dateSelected() {
+    this.messages.push('Date has changed. ');
+  }
+}
+
+
+@Component( {
+  selector: 'my-calendar',
+  outputs: ['selectedChange'],
+  template: `
+    <div *ngFor="let date of dates">
+      <button (click)="_selected(date)">{{date}}}</button>
+    </div>
+    <div>Date: {{selected}}</div>
```

Comment:
probably put this outside of the calendar

Reply:
How should we go about doing this? And what is our goal for achieving this? If we do put this outside of the calendar (in the `CdkDatepickerOverviewExample`), I think we would have to create an instance of `MyCalendar` in the `CdkDatepickerOverviewExample` component to get the `selected` property and show it. If we were to put this in `CdkDatepicker`, I think we would have to go within the component's selector and put it in its template.

Reply:
Also, is our goal to make this `div` specifically separated from `MyCalendar`?